src/Editor/Editor.cpp

```diff
@@ -533,22 +533,19 @@ Entity* Editor::GetCamera() const {
 
 void Editor::Play() {
     Resources().Save();
-    editorState = Hymn().ToJson();
+    sceneState = Hymn().world.GetSaveJson();
```

Comment:
Now the scene is saved but the Hymn isn't (Hymn includes eg. a list of all scripts). Ideally both would be saved rather than just one.

Reply:
Can the hymn be altered when you play?

Reply:
No.

Reply:
So why should we save it aswell?